docs/CHANGELOG.asciidoc

```diff
@@ -51,11 +52,476 @@ Review important information about the {kib} 7.12.x releases.
 // * <<release-notes-7.0.0-alpha1>>
 
 --
+
+[[release-notes-7.13.0]]
+== {kib} 7.13.0
+
+coming::[7.13.0]
+
+For information about the {kib} 7.13.0 release, review the following information.
+
+[float]
+[[breaking-changes-7.13.0]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and performance.
+Before you upgrade to 7.13.0, review the breaking changes, then mitigate the impact to your application.
+
+// tag::notable-breaking-changes[]
+
+[discrete]
+[[breaking-97206]]
+.Remove Elastic Agent routes and related services
+[%collapsible]
+====
+*Details* +
+You are now unable to run and manage Elastic Agent with {kib}. For more information, refer to {kibana-pull}97206[#97206].
```

Comment:
This description is potentially misleading because the Fleet UI lives within Kibana. The main point here is that Elastic Agents will no longer communicate with Kibana to enroll agents, get agent policies, collect status info, and so on. Instead, the agents will use Fleet Server as a control plane. It might be worth pointing to the Fleet Server docs.

The description here makes it sound like the user is _losing_ something when really they are actually gaining scalability. :-) 

@mostlyjason Can you provide more guidance on how to phrase this?